sample/common/src/jvmMain/kotlin/com/example/sqldelight/hockey/data/Db.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.sqldelight.hockey.data
+
+import com.example.sqldelight.hockey.HockeyDb
+import com.squareup.sqldelight.db.SqlDriver
+
+object Db {
```

Comment:
might be a way to share some of the jvm/ios `Db` in the common module, but can be in a different PR